classic stock enhancment

JadeOfMaar opened the pull request:
* upgraded resource distributions
* ensured rock has a definition
* defined Hydrokarbon and RHK-1
* patch for stock scanners and scansat
* amended kit descriptions
* added colors for mined resources in stock orbital survey

JadeOfMaar committed:
classic stock enhancment

* upgraded resource distributions
* ensured rock has a definition
* defined Hydrokarbon and RHK-1
* patch for stock scanners and scansat
* amended kit descriptions

JadeOfMaar committed:
ordered resources in scanners

JadeOfMaar committed:
adjusted default resource distributions

JadeOfMaar committed:
indentations, saw an NRE issue coming

Angel-125 commented on GameData/WildBlueIndustries/000WildBlueTools/Templates/ClassicStock/Storage/Hydrokarbon.txt:
```diff
@@ -0,0 +1,354 @@
+
+RESOURCE_DEFINITION
+{
+	name = RHK1 // Karbonite
```
Can we get rid of references to Karbonite? I'd prefer to reduce the amount of finger pointing if people get upset

Angel-125 commented on GameData/WildBlueIndustries/000WildBlueTools/Templates/ClassicStock/Storage/Hydrokarbon.txt:
```diff
@@ -0,0 +1,354 @@
+{
+	name = Hydrokarbon
+	author = JadeOfMaar
+	shortName = Hydrokarbon
```
Short names are no longer needed. Please use title instead if you need a name that's not the actual resource name.

JadeOfMaar committed:
adjustments

* fixed base volumes
* added exo atmo to atmo scanner
* removed references and shortname param

JadeOfMaar committed:
fixed global :vs: planetary distribution

JadeOfMaar committed:
scanner patch update

finished off SCANsat patch (tiny issue: overlay palette options are
narrow now); reordered Rock in stock scanner
